ledger/complete/wal/wal.go

```diff
@@ -16,6 +16,7 @@ const SegmentSize = 32 * 1024 * 1024
 
 type LedgerWAL struct {
 	wal            *prometheusWAL.WAL
+	paused         bool
```

Comment:
not thread safe, but i think that's okay.

Reply:
yeah I think wal is not also threadsafe.